File: games/friendzone4/src/main/java/fr/purpletear/friendzone4/game/activities/main/MainInterface.java
package fr.purpletear.friendzone4.game.activities.main;

public interface MainInterface {
    void onClickChoice(Phrase p);
    void onInsertPhrase(int position, boolean isSmoothScroll);
    void onClickContact(String code, Contact.Type type);
    void onClickItem(String type, String name);
    void onMissedChoice(Phrase p);
    void onTouchJoystick();
    void onReleaseJoystick();
    void onJoystickInfoHit(long startMs, String code);
}

File: games/friendzone2/src/main/java/fr/purpletear/friendzone2/activities/main/MainInterface.kt
/*
 * Copyright (C) PurpleTear, Inc - All Rights Reserved
 * Unauthorized copying of this file, via any medium is strictly prohibited
 * Proprietary and confidential
 * Written by Hocine Belbouab <hbdeveloppeur@gmail.com>, 1/10/19 11:38 AM
 *
 */

package fr.purpletear.friendzone2.activities.main

import fr.purpletear.friendzone2.configs.Phrase

interface MainInterface {
    fun onClickChoice(p: Phrase)
    fun onInsertPhrase(position: Int, isSmoothScroll: Boolean)
    fun onClickSound(name : String)
}
